src/components/organisms/EarthEditor/CanvasArea/hooks.ts

```diff
@@ -140,6 +149,39 @@ export default (isBuilt?: boolean) => {
     document.title = title;
   }, [isBuilt, title]);
 
+  const [updateLayerLatLng] = useUpdatePropertyValueLatLngMutation();
+
+  const handleDragLayer = useCallback((_layerId: string, _position: LatLngHeight | undefined) => {
```

Comment:
Let's keep style as consistent as we can!
```suggestion
  const handleDragLayer = useCallback((_layerId: string, _position?: LatLngHeight) => {
```